reflex/utils/processes.py

```diff
@@ -145,21 +145,34 @@ def new_process(args, run: bool = False, show_logs: bool = False, **kwargs):
     return fn(args, **kwargs)
 
 
-def run_concurrently(*fns: Union[Callable, Tuple]):
+@contextlib.contextmanager
+def run_concurrently(
+    *fns: Union[Callable, Tuple]
```

Comment:
Should we add a param indicating if they want to return a generator? I think in the other cases, it will make them do an `with` and `pass`. Maybe the generator option should be the opt-in with the param?

Reply:
i don't think it's possible to have a function that returns normally OR is a generator; you kind of have to pick one or the other.

what i can do though is rename this function as `run_concurrently_context`, and then make a new function with the same name `run_concurrently` that just does the `with ...: pass` internally.

what do you think of that?

personally I don't have a problem if this function is always a generator, but keeping the original name as a normal function would result in less collateral changes to unrelated parts of the code.

Reply:
I think this works great